src/main/java/net/discordjug/javabot/systems/staff_commands/RunScheduledTaskCommand.java

```diff
@@ -0,0 +1,81 @@
+package net.discordjug.javabot.systems.staff_commands;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.scheduling.config.ScheduledTaskHolder;
+
+import net.discordjug.javabot.data.config.BotConfig;
+import net.discordjug.javabot.util.Responses;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.events.interaction.command.CommandAutoCompleteInteractionEvent;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.AutoCompleteQuery;
+import net.dv8tion.jda.api.interactions.commands.Command;
+import net.dv8tion.jda.api.interactions.commands.Command.Choice;
+import net.dv8tion.jda.api.interactions.commands.DefaultMemberPermissions;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.Commands;
+import xyz.dynxsty.dih4jda.interactions.AutoCompletable;
+import xyz.dynxsty.dih4jda.interactions.commands.application.SlashCommand;
+import xyz.dynxsty.dih4jda.util.AutoCompleteUtils;
+
+/**
+ * This command allows manually executing scheduled tasks.
+ */
+public class RunScheduledTaskCommand extends SlashCommand implements AutoCompletable{
+	
+	private ScheduledTaskHolder taskHolder;
+	
+	/**
+	 * The constructor of this class, which sets the corresponding {@link SubcommandData}.
+	 * @param botConfig the configuration of the bot
+	 * @param taskHolder A Spring object managing scheduled tasks
+	 */
+	public RunScheduledTaskCommand(BotConfig botConfig, ScheduledTaskHolder taskHolder) {
+		setCommandData(Commands.slash("run-task", "(ADMIN ONLY) Run scheduled tasks")
+				.setDefaultPermissions(DefaultMemberPermissions.enabledFor(Permission.ADMINISTRATOR))
+				.setGuildOnly(true)
+				.addOption(OptionType.STRING, "name", "Class name of the task", true, true)
+		);
+		setRequiredUsers(botConfig.getSystems().getAdminConfig().getAdminUsers());
+		
+		this.taskHolder = taskHolder;
+		
+		
+	}
+	
+	@Override
+	public void execute(SlashCommandInteractionEvent event) {
+		String name = event.getOption("name").getAsString();
```

Comment:
isnt this missing a null-check?

Reply:
The option is set to be required but I can still add a null-check.

Reply:
yup

Reply:
done